.github/workflows/pytorch-version-tests.yml

```diff
@@ -0,0 +1,42 @@
+on: 
+  schedule:
+    # Run at 00:00 UTC on Sunday
+    - cron:  '0 0 * * 0'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 10
+      fail-fast: false
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
```

Comment:
I think 3.8 support was not available for <1.4
And we can remove pytorch 1.5 too